src/plotconfig.jl

```diff
@@ -213,6 +213,7 @@ function PlotConfig(T::Val{:erp})
             xtickformat = "{:.1f}",
         ),
         colorbar = (; label = "", flipaxis = true),
+        visual = (; colormap = :roma, color = Makie.wong_colors()),
```

Comment:
do we really need this? `color` specifies only the Makie defaults anyway, so then you could never overwrite them with `with_theme` which would be a shame

Reply:
okay, that actually happening. Where i can catch color form with_theme in the function?

Reply:
why do you need to catch it?

Reply:
to be able to prioritize it in the code, for instance

Reply:
I dont understand your answer. What is lost if we simply remove the color here? Then users could set it with `with_theme`